phases/phase-1/challenges/referred/wjdfx

```diff
@@ -1 +1,2 @@
 Ping
+iammelea
```

Comment:
I'm sorry, you don't need to put your name here. 

What you have to do is the following: 

```
echo "wjdfw" >> ./phases/phase-1/challenges/referred/iammelea
```

After doing so, please delete this line from the file.